CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+# Contributing
+
+Thank you for considering making contributions to Ethermint! Start by taking a look at this [coding repo](https://github.com/tendermint/coding) for overall information on repository workflow and standards.
+
+Please follow standard github best practices: fork the repo, branch from the tip of develop, make some commits, and submit a pull request to develop. See the [open issues](https://github.com/cosmos/ethermint/issues) for things we need help with!
+
+Please make sure to use `gofmt` before every commit - the easiest way to do this is have your editor run it for you upon saving a file. Additionally please ensure that your code is lint compliant by running `make lint`
+
+Looking for a good place to start contributing? How about checking out some [good first issues](https://github.com/cosmos/ethermint/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
+
+## Forking
+
+Please note that Go requires code to live under absolute paths, which complicates forking.
+While your fork should live at `https://github.com/<username>/ethermint`,
+the code should not exist at  `$GOPATH/src/github.com/<username>/ethermint`.
+Instead, you should use `git remote` to add the fork as a new remote for the original repo,
+`$GOPATH/src/github.com/cosmos/ethermint `, and do all the work there.
+
+For instance, to create a fork and work on a branch of it, One would:
+
+  * Create the fork on github, using the fork button.
+  * Go to the original repo checked out locally (i.e. `$GOPATH/src/github.com/cosmos/ethermint`)
+  * `git remote rename origin upstream`
+  * `git remote add origin git@github.com:<username>/ethermint.git`
+
+Now `origin` refers to my fork and `upstream` refers to the Cosmos-SDK version.
+So I can `git push -u origin <branch>` to update my fork, and make pull requests to Cosmos-SDK from there.
+Of course, replace `<username>` with your git handle.
+
+To pull in updates from the origin repo, run
+
+    * `git fetch upstream`
+    * `git rebase upstream/develop` (or whatever branch you want)
+
+Do not make Pull Requests to `master`, they will not be considered.
+
+## Dependencies
+
+We use [`dep`](https://github.com/golang/dep) to manage dependencies.
+
+That said, the master branch of every Cosmos repository should just build
+with `go get`, which means they should be kept up-to-date with their
+dependencies so we can get away with telling people they can just `go get` our
+software.
+
+Since some dependencies are not under our control, a third party may break our
+build, in which case we can fall back on `dep ensure` (or `make
+get_vendor_deps`). Even for dependencies under our control, `dep` helps us to
```

Comment:
I'll be changing this to `get-vendor-deps` in ethermint 👍

Reply:
Awesome changed here.